### DISCUSSION TITLE
every video returns "There was an error with the server"

### DISCUSSION THREAD

**Discussion by mojii100** [ReVanced/revanced-discussions#1124] [Question]
Trying to play any video returns "there was an error with the server [400] tap to retry". Tapping to retry and closing and reopening the app loads the video for a second and then goes back to error 
![Screenshot_20241217-184612](https://github.com/user-attachments/assets/f5cab85a-78c8-46f7-987a-7950f4e71e7d)
I tried to search for the error message in here but couldn't find anyone else reporting it

**Comment by oSumAtrIX** [comment]
Update by repatching.
